src/components/Flashcard/CardContent.js

```diff
@@ -1,45 +1,77 @@
 import React, { useState, useEffect } from 'react';
 import styles from './Flashcard.module.css';
-import { FaQuestion, FaCommentDots, FaSave, FaCheck, FaTimes } from 'react-icons/fa';
 
-import { Button } from 'components';
+import {
+    Button,
+    CrossIcon,
+    CheckIcon,
+    SaveIcon,
+    AnswerIcon,
+    QuestionIcon,
+} from 'components';
 
-export function CardContent({ question, addToUserBase, known, unknown}) {
-    const [isKnownAdded, setIsKnownAdded] = useState('iconButton');
-    const [isUnknownAdded, setIsUnknownAdded] = useState('iconButton');
+export function CardContent({ question, addToUserBase, known, unknown }) {
+    const [isKnownAdded, setIsKnownAdded] = useState('#416071');
```

Comment:
these initial states don't tell much, what are they? name of the state suggest the initial state to be true or false